### ISSUE TITLE
[Figuring out] Are decorators supported by Apollo?

### ISSUE THREAD

**Issue by RomanGalochkin** [foretold-app/foretold#643]
A simplest experiment

**Comment by RomanGalochkin** [comment]
I have found that the directives are not what we are looking forward to. But instead of this we can use simple arguments:
```
query {
  measurement(id: "9ffa7041-aa3b-44ae-b72f-8531c2336711") {
    id
    value {
      floatCdf (truncate: 6 round: 2) {
        xs
        ys
      }
    }
  }
}
```

![Screenshot_20190906_094544](https://user-images.githubusercontent.com/16107421/64406683-19990100-d08b-11e9-82c6-a882d54906e1.png)

And it seems that it works as we expected. That server response became much more less. And what I have found else is that we need to optimize Queries. For instance we do not need "measurement.value" when we query "scores". And the same for another scoring, so we need optimize Queries first of all (it won't take much time). So before:

![Screenshot_20190906_094855](https://user-images.githubusercontent.com/16107421/64406853-8c09e100-d08b-11e9-94b1-b936d953dfe9.png)
1.12 s is here

and after with optimized graphql query:
![Screenshot_20190906_095150](https://user-images.githubusercontent.com/16107421/64407016-f3c02c00-d08b-11e9-922f-93cf66fb4427.png)
0.78s.

So it has a sense.
